src/coffee/directives/api/models/parent/base-polys-parent-model.coffee

```diff
@@ -134,8 +113,9 @@ angular.module('uiGmapgoogle-maps.directives.api.models.parent')
               .then (state) =>
                 payload = state
                 if(payload.updates.length)
-                  #TODO: not supporting updates yet
-                  $log.info("polygons updates: #{payload.updates.length} will be missed")
+                  _async.each payload.updates, (obj) =>
+                    _.extend obj.child.scope, obj.model
```

Comment:
This is where setVisible and other attributes were getting borked. As the child.scope never had the correct values.